lib/exercism/acl.rb

```diff
@@ -1,7 +1,11 @@
 class ACL < ActiveRecord::Base
   def self.authorize(user, problem)
-    ACL.create(user_id: user.id, language: problem.track_id, slug: problem.slug)
-  rescue ActiveRecord::RecordNotUnique
-    ACL.where(user_id: user.id, language: problem.track_id, slug: problem.slug).update_all(updated_at: Time.now.utc)
+    acl = ACL.where(user_id: user.id, language: problem.track_id, slug: problem.slug).first
+
+    unless acl
```

Comment:
I'd like to avoid `unless ... else` because the else then becomes a double negative. Can this be turned around to make sense with an `if`?